components/ConditionSetDesigner/ConditionSetItem.vue

```diff
@@ -0,0 +1,202 @@
+                if (x !== -1) {
+                    return Array.isArray(placeholders[clearedKey])
+                        ? getValuesByKeys(parameters[x].options, placeholders[clearedKey])
+                        : getValueByKey(parameters[x].options, placeholders[clearedKey]);
+                }
+                return placeholders[clearedKey] || placeholder;
+            });
+        },
+    },
+    methods: {
+        ...mapActions('conditions', [
+            'setConditionValue',
+        ]),
+        getComponentViaType(type) {
+            switch (type) {
+            case 'SELECT':
+            case 'MULTI_SELECT':
+                return Select;
+            case 'NUMERIC':
+            case 'TEXT':
+            case 'UNIT':
+            case 'PRICE':
+                return TextField;
+            default:
+                return null;
+            }
+        },
+        hasOptions(type, options = {}) {
+            return type === 'SELECT' || type === 'MULTI_SELECT'
+                ? Object.values(options)
+                : [];
+        },
+        setConditionValueByType({
+            value, name, type, options,
+        }) {
+            let tmpValue = value;
+            if (type === 'SELECT' || type === 'MULTI_SELECT') {
+                tmpValue = Array.isArray(value)
+                    ? getKeysByValues(options, value)
+                    : getKeyByValue(options, value) || null;
+            }
+            this.setConditionValue({
+                conditionId: this.itemId,
+                parameterName: name,
+                parameterValue: tmpValue,
+            });
+        },
+        getConditionValue({ name, type, options }) {
+            const condition = this.conditionsValues[this.itemId];
+
+            if (!condition) return '';
+            if (type === 'SELECT' || type === 'MULTI_SELECT') {
+                return Array.isArray(condition[name])
+                    ? getValuesByKeys(options, condition[name])
+                    : getValueByKey(options, condition[name]);
+            }
+            return condition[name] || '';
+        },
+        conditionParametersAreValidate(index, key) {
+            return this.validationErrorsAreNotNull
+            && this.validationErrorsHasProperty(index)
+            && this.validationErrors[index][key]
+                ? this.validationErrors[index][key].join(', ') : null;
+        },
+        errorParamsMessage(key) {
+            const parametersIndex = `conditions_element-${this.itemRow}`;
+            return this.conditionParametersAreValidate(parametersIndex, key);
+        },
+    },
+};
+</script>
+
+<style lang="scss" scoped>
+    .condition {
+        display: flex;
+        flex-direction: column;
+        height: 100%;
+        border: 1px solid $grey;
+        background-color: $background;
+        cursor: move;
+        overflow: hidden;
+
+        &::after {
+            position: absolute;
```

Comment:
By centering elements with that way you will lose the sharpness of element. It will lose the quality of text